package.json

```diff
@@ -36,21 +37,22 @@
     "@babel/preset-typescript": "7.24.7",
     "@changesets/changelog-github": "^0.5.0",
     "@changesets/cli": "^2.27.7",
-    "@types/jest": "27.4.1",
+    "@jest/test-utils": "^0.0.0",
```

Comment:
Why was `@jest/test-utils` added?

Reply:
Turns I didnt need it and have removed it. Was trying to figure out how to get Jest to support comparing objects with `*[Symbol.iterator]` with `.toBeEqual`. It ended up just needing a version bump of jest